docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -443,8 +532,30 @@ the :ref:`dronecan_fc_tutorial` tutorial.
 
 Arming and Arming Bypass
 ================================
+
+Standard Arming
+*****************
 DroneCAN can use the same advanced arming procedure as all other throttle sources. The details of this arming procedure are covered in the :ref:`manual_advanced_arming` section.
 
+Arming with ArmingStatus
+******************************
+Vertiq modules can arm and disarm based off of DroneCAN's :ref:`ArmingStatus message <arming_status>`. This means that whenever your flight controller broadcasts an ArmingStatus 
+``STATUS_FULLY_ARMED`` message, your module will transition from disarmed to armed, or if already armed, will remain armed. These transitions are not subject to the constraints 
+set by your :ref:`arming throttle region <arming_throttle_regions>`, so any ``STATUS_FULLY_ARMED`` message will arm your module. When your flight controller broadcasts a ``STATUS_DISARMED`` message, your module 
+will similarly transition from armed to disarmed.
+
+Whether your module arms via the ArmingStatus message is configured through the ``arming_by_arming_status`` parameter in the :ref:`DroneCAN node client <uavcan_node>`.
+
+An important note is that the ArmingStatus message can be used to arm a vehicle controlled by a different protocol. For example, if your vehicle has connections for both DroneCAN and 
+:ref:`PWM <hobby_protocol>` control, your module can arm with DroneCAN, but be controlled by PWM so long as :ref:`manual_arming_throttle_source` is configured to ``Hobby``.
+
+Further, we highly recommend disabling your module's ability to :ref:`arm on throttle <arming_throttle_regions>`. Suppose your module is configured to arm on throttle as well as arm on 
```

Comment:
I may just add the caveat at the end of this first sentence "...if you are using the ArmingStatus to arm". To avoid it seeming like a blanket recommendation to not use throttle. I know it should be obvious from context, just want to be ultra specific for our customers

Also, you recommend turning off arming on throttle, but you don't mention disarming on throttle. One should imply the other, but again, super specificity is good.